packages/sqlite-storage/package.json

```diff
@@ -58,17 +58,17 @@
     "@babel/core": "^7.20.0",
     "@babel/preset-env": "^7.20.0",
     "@babel/runtime": "^7.20.0",
-    "@react-native/babel-preset": "^0.73.0",
-    "@rnx-kit/metro-config": "^1.3.14",
+    "@react-native/babel-preset": "^0.75.0",
+    "@rnx-kit/metro-config": "^2.0.0",
     "@types/jest": "^29.5.5",
     "@types/react": "^18.2.44",
     "eslint": "^8.51.0",
-    "jest": "^29.7.0",
+    "jest": "^29.2.1",
```

Comment:
This looks like a downgrade, but isn't:

![image](https://github.com/user-attachments/assets/400f55e4-da8f-4b26-a584-cfa5cdba0271)

I just ran `npx --package @rnx-kit/align-deps@latest rnx-align-deps --requirements react-native@0.75 --write` to update everything in one go.